coding_style/README.adoc

```diff
@@ -150,6 +150,16 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   But do use quotes for user-side strings such as descriptions, names, and messages.
 * Even if JSON is valid YAML and Ansible understands it, do only use JSON syntax if it makes sense (e.g. a variable file automatically generated) or adds to the readability.
   In doubt, nobody expects JSON so stick to YAML.
+* A single jinja template should not exceed 250 lines in length
+* The aggregate number of jinja includes should not exceed 3
+* Jinja templates should not be used to create structured data but instead text and semi-structured data
+* Filter plugins are preferred over the use of data manipulation or transformation in Jinja templates when the necessary data transformation cannot be accomplished with existing and available filter plugins or the transformation of data is unique to the use case.
```

Comment:
The sentence is difficult to understand, I think you mean actually two things:

1. prefer filter plugins to Jinja2 templates
2. for more complex use cases, write a custom filter plugin

Anyway you meant it, split the sentence somehow, I think it'll help.